test/rfq_blockchain_utils_test.ts

```diff
@@ -272,6 +279,38 @@ describe(SUITE_NAME, () => {
             expect(txHash).to.match(/^0x[0-9a-fA-F]+/);
         });
     });
```

Comment:
Can we add a test for `transformTxDataToTransactionRequestAsync`? Some logic in there that could do well having a test around:
- The fallback to address being the exchangeProxy
- The logic regarding mapping some of the fields
- The ethers fallback (not necessary to test this imo, but nice for completion - also see comment about possibly moving this logic out of this method)

Reply:
good call done